### PR TITLE
Heatmap

### PR DESCRIPTION
**Pull request by thv92**
Made a toggle for switching the map from heatmap style back to the default style and vice versa.

Signed-off-by: Thien Vo <vohuythien@utexas.edu>

### CODE REVIEW

**Comment by jdk2pq** [comment]
Can you remove the .DS_Store files from this pull request? They are automatically generated files from a Mac, and they shouldn't be checked in to the repository.

File: geomesa-ui/src/main/ui/package.json [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
+    "lodash": "~3.3.1",
     "eyes": "~0.1.8",
-    "grunt": "~0.4.2",
+    "grunt": "~0.4.5",
     "matchdep": "~0.3.0",
-    "mocha": "~1.17.1",
-    "requirejs": "~2.1.11",
-    "karma": "0.10.9",
+    "mocha": "~2.1.0",
+    "requirejs": "~2.1.16",
```
Comment (jnh5y) [inline geomesa-ui/src/main/ui/package.json]: Not a big deal, but I was liking the alphabetical order list.

**Comment by jnh5y** [comment]
@thv92 We can add .DS_Store to the root .gitignore.

**Comment by thv92** [comment]
I've alphabetized and added .DS_Store to the root .gitignore and also the .gitignore inside geomesa_ui.

File: geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/CompositeConverter.scala [hunk 2]
```diff
@@ -54,7 +52,7 @@ class CompositeConverter[I](val targetSFT: SimpleFeatureType,
 
   implicit val ec = new EvaluationContext(Map(), Array())
   def processIfValid(input: I, pred: Predicate, conv: SimpleFeatureConverter[I]) =
-    if(pred.eval(input)) conv.processSingleInput(input) else None
+    Try { pred.eval(input) }.toOption.flatMap { v => if(v) conv.processSingleInput(input) else None }
```
Comment (apc3n) [inline geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/CompositeConverter.scala]: catch the exception you expect to occur.. toOption will turn any Failure into a None. For example you probably don't want to turn a NPE into a None.. you want to throw that particular exception.

**Pull request closed without merging**